website/src/components/SearchPage/SeqPreviewModal.tsx

```diff
@@ -135,3 +133,29 @@ export const SeqPreviewModal: React.FC<SeqPreviewModalProps> = ({
         </Transition>
     );
 };
+
+interface DownloadButtonProps {
+    seqId: string;
+}
+
+const DownloadButton: React.FC<DownloadButtonProps> = ({ seqId }: { seqId: string }) => {
+    return (
+        <div className='dropdown dropdown-hover relative inline-block'>
+            <button className={BUTTONCLASS}>
+                <IcBaselineDownload className='w-6 h-6' />
+            </button>
+            <ul className='dropdown-content z-20 menu p-1 shadow bg-base-100 rounded-btn absolute top-full -left-4'>
+                <li>
+                    <a href={routes.sequencesFastaPage(seqId, true)} className='block px-4 py-2 hover:bg-gray-100'>
+                        FASTA
```

Comment:
```suggestion
                        Download FASTA
```

IMO button texts should indicate an action - in the case of actionny buttons (we could make things wider to fit this) 

Same for below